### PR TITLE
Slider & RangeSlider: correct event parameter should be passed to valueChanged event(T926119)

### PR DESCRIPTION
**Pull request by ksercs**
…

### CODE REVIEW

File: js/ui/slider/ui.slider.js [hunk 1]
```diff
@@ -70,36 +70,42 @@ const Slider = TrackBar.inherit({
             leftArrow: function(e) {
                 e.preventDefault();
                 e.stopPropagation();
+                this._saveValueChangeEvent(e);
```
Comment (DokaRus) [inline js/ui/slider/ui.slider.js]: Let's extract duplicated code

**Review by alexander-kotov-dx — DISMISSED** [review]

File: js/ui/slider/ui.slider.js (hunk 1 above)
Comment (ksercs) [reply js/ui/slider/ui.slider.js]: fixed

File: js/ui/slider/ui.slider.js [hunk 2]
```diff
@@ -68,44 +68,46 @@ const Slider = TrackBar.inherit({
...
-                e.stopPropagation();
+                this._processKeyBoardEvent(e);
+
                 const max = this.option('max');
                 this.option('value', max);
             }
         });
     },
 
+    _processKeyBoardEvent: function(e) {
```
Comment (San4es) [inline js/ui/slider/ui.slider.js]:
```suggestion
    _processKeyboardEvent: function(e) {
```

Comment (ksercs) [reply js/ui/slider/ui.slider.js]: fixed

**Review by DokaRus — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Review by San4es — COMMENTED** [review]
👍

**Pull request merged**